simulator/risc_v/riscv_instr.cpp

```diff
@@ -310,7 +312,7 @@ const auto& find_entry( std::string_view name)
     for (const auto& e : cmd_desc<I>)
         if ( e.entry.name == name)
             return e;
-        
+
```

Comment:
You are the third person who removes trailing spaces here.
Could you please explain what motivated you to do that?

Reply:
This happened reflexively. I can not explain.